sdc/runtests.py

```diff
@@ -62,8 +62,7 @@ def load_tests(loader, tests, pattern):
 
 
 if __name__ == '__main__':
-    if sdc.config.config_pipeline_hpat_default:
-        # initialize MPI
-        get_rank()
+    # initialize MPI to avoid "Attempting to use an MPI routine before initializing MPICH" in any pipeline
```

Comment:
@densmirn Why there's a need of MPI in numba pipeline at all? Is this some kind of w/a?